README.md

```diff
@@ -1,80 +1,69 @@
-# Face Recognize
-
-Blocks for working with facial recognition and images
-
-Get Encoding From File
-========
-Find a face encoding from an image file, send encoding and other data as signal
+CaptureFrame
+============
+Grab a frame of video from a specified camera and send the frame data as a signal.
 
 Properties
---------------
-- Image Path: Full path to the image file that will be added
-- User ID: Id of the face being added to the database
-- Save Name: Name of the face being added to the database
+----------
+- **camera**: Where to find the local camera to read from.
+- **ipcam**: Whether or not to use an IP Camera.
+- **ipcam_address**: Address for where to find the IP camera.
 
-Dependencies
-----------------
-- face_recognition
-- pickle
-- base64
-
-Input
--------
-- Any signal to trigger the block to run
+Inputs
+------
+- **default**: Any signal to trigger a frame being grabbed from the specified camera.
 
-Output
+Outputs
 -------
-- A signal containing the facial encoding, user id, and name
-
-sample output:
-
-```
-{
- 'encoding': 'gANjbnVtcHkuY29yZS5tdWx0aWFycmF5Cl9yZWNvbn...',
- 'name': 'Barack',
- 'user_id': 'bobama'
-}
-```
+- **default**: A signal containing the serialized and stringified video frame.
 
-Capture Frame
-=============
-Grab a frame of video from a specified camera and send the frame data as a signal.
+Commands
+--------
 
 Dependencies
-----------------
-- opencv-python
-- pickle
+------------
 - base64
```

Comment:
There are a few dependencies here that are builtins, `base64`, `pickle`, and `urllib.requests` (I think that was meant to be `urllib.request` anyways). Also get rid of these in requirements.txt